app/templates/discover.hbs

```diff
@@ -26,7 +26,7 @@
         </div>
     </div>
     {{#if queryBody}}
-        <div class="row" style="text-align: center; font-size:75%; margin-top:15px">
+        <div class="row hidden-xs" style="text-align: center; font-size:75%; margin-top:15px">
```

Comment:
I can't think of a way the user could see both the filter facets and the results at the same time -- the screen just isn't large enough. Although I agree that features shouldn't be hidden.